examples/gcoap/gcoap_cli.c

```diff
@@ -241,15 +242,21 @@ static size_t _send(uint8_t *buf, size_t len, char *addr_str, char *port_str)
         }
         remote.netif = pid;
     }
+#else
+    remote.netif = SOCK_ADDR_ANY_NETIF;
+#endif
+
     /* parse destination address */
     if (ipv6_addr_from_str(&addr, addr_str) == NULL) {
         puts("gcoap_cli: unable to parse destination address");
         return 0;
     }
+#ifdef MODULE_GNRC_SOCK_UDP
```

Comment:
```suggestion
    if (IS_USED(MODULE_GNRC_SOCK_UDP)) {
```

maybe merge with `if` below.